teuthology/orchestra/run.py

```diff
@@ -395,7 +395,8 @@ def run(
     timeout=None,
     cwd=None,
     # omit_sudo is used by vstart_runner.py
-    omit_sudo=False
+    omit_sudo=False,
+    shell=True
```

Comment:
Needs docs below.

I wonder if it'd be better to do:

``**kwargs``

instead so we don't need to keep them in sync all the time?

Reply:
> Needs docs below.
> 

Added docs.

> I wonder if it'd be better to do:
> 
> `**kwargs`
> 
> instead so we don't need to keep them in sync all the time?

I personally feel accepting arguments in the current fashion keeps the API easier to get a grip on and also keeps track of parameters that are acceptable.

Reply:
We should stop doing this, like adding useless parameters, and I was hopping we can get rid of `omit_sudo` as far as we added file operations api with _sudo_ arguments. The vstart_runner.py should not *hack* teuthology, but it should use it in the correct way.

Reply:
As argument, the usin of shell=True is a feature already, otherwise multiline script commands will fail once trying to run in this mode.